addons/suggest_locations/controllers/controllers.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from odoo import http, _
+from odoo.http import request
+
+
+class PutawayStrategy(http.Controller):
+    def _process_suggested_inputs(self, move_line_ids, limit=30, **kwargs):
```

Comment:
Do we need this limit here ? there is already one limit at `suggest_locations()`.

Also we are not using `kwargs`, that should probably be `values`.